.github/workflows/lint.yml

```diff
@@ -60,3 +60,23 @@ jobs:
 
         export LANG=en-us
         make pylint
+  
+  pylint_site_packages:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Lint with pylint
+      run: |
+        sudo apt-get install libkrb5-dev
+        pip install -r requirements/devel.txt
+
+        export LANG=en-us
```

Comment:
probably we don't need the LANG variable here. If it works without it please remove it.